added visualbar to similarity column (#184)
ServiceNow/azimuth: CHANGELOG.md

### Added
- Rows of Performance Analysis table now link to exploration page with filters applied.

> - Added a distribution bar for the similarity column in the semantically similar utterances.

Reviewer: It's not really a "distribution bar". We called it a "visual bar" in the previous release, so for consistency I suggest
```suggestion
- Added visual bars to the similarity column in the semantically similar utterances.
```
